src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -203,6 +203,32 @@ public void newUserCreateEditMessageWallAndRenameDone() {
 
     Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
     wall.verifyMessageEditTextRenameDone(title, message, newName);
+
+
+//
+//    wall.open(newName);
+//    String encodedUrl = renameUserPage.encodeToURL(newName);
+//
+//    String expectedWallUrl = String.format("%s/wiki/Message_Wall:%s", urlBuilder.getUrlForWiki(),
+//                                           encodedUrl);
+//
+//    Assertion.assertEquals(driver.getCurrentUrl(), expectedWallUrl);
+//    wall.verifyMessageEditTextRenameDone(title, message, newName);
+//    String
+//        theContent =
+//        EmailUtils.getFirstEmailContent(credentials.emailAnonymousUserTestWikia,
+//                                        credentials.emailAnonymousUserTestWikiaPassword,
+//                                        "Your username change on FANDOM is complete!");
+//  }
+//
+//  @Test
+//  public void anonyizationBlogTest() {
+//
+//    Credentials credentials = new Credentials();
+//    EmailUtils.deleteAllEmails(credentials.emailAnonymousUserTestWikia,
+//                               credentials.emailAnonymousUserTestWikiaPassword);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)